apps/frontend/src/components/keplerGl/index.ts

```diff
@@ -13,12 +13,14 @@ import {
   replacePanelTab,
   replaceTooltipConfig,
 } from './factories';
+import { replacePanelTitle } from './factories/PanelTitleFactory';
```

Comment:
You should import this as the other replace functions.
To do so, you must re-export each function in the `factories/index.ts` file.
It's known as a barrel file : https://basarat.gitbook.io/typescript/main-1/barrel